server/src/main/java/org/jetbrains/bsp/bazel/server/sync/languages/java/JavaModule.java

```diff
@@ -107,4 +110,20 @@ public int hashCode() {
         sourcesClasspath,
         ideClasspath);
   }
+
+  @Override
+  public String toString() {
```

Comment:
this this could be generated using intellij, right?

Reply:
Default intellij generated to string is not readable enough if you want to actually understand something. This approach is farily custom, I specify how to render some sequences (long or short). Maybe it could be automated more.